src/Models/Containers/Column.ts

```diff
@@ -21,7 +21,12 @@ export class ColumnModel extends ScopeModel {
         this.style = StringUtils.normalize(json.style);
         this.height = StringUtils.normalize(json.height);
         this.verticalContentAlignment = StringUtils.normalize(json.verticalContentAlignment, 'top');
-        this.width = StringUtils.normalize(json.width);
+
+        let columnWidth = StringUtils.normalize(StringUtils.parseNumber(json.width));
+        if (typeof columnWidth === 'number') {
+            columnWidth = columnWidth < 0 ? 0 : columnWidth;
+        }
+        this.width = columnWidth;
```

Comment:
Have you test this change?

Reply:
Yes. change width property to see render result.